src/routes/(app)/pos/touch/+page.svelte

```diff
@@ -32,11 +36,29 @@
 			<a class="col-span-2 touchScreen-category-cta" href="?filter=all">TOUS LES ARTICLES</a>
 
 			<div class="col-span-2 grid grid-cols-2 gap-4">
-				{#each productFiltered as product}
+				{#each displayedProducts as product}
 					{#if !isPreorder(product.availableDate, product.preorder)}
 						<ProductWidgetPOS {product} pictures={picturesByProduct[product._id]} />
 					{/if}
 				{/each}
+				<div class="col-span-2 grid-cols-1 flex gap-2 justify-center">
+					{#if next > 0}
+						<a
+							class="btn touchScreen-product-secondaryCTA"
+							on:click={() => (next = Math.max(0, next - POS_PRODUCT_PAGINATION))}
+							href={`?filter=${filter}&skip=${Math.max(0, next - POS_PRODUCT_PAGINATION)}`}
+							>&#9664;</a
```

Comment:
Let's use an emoji, unicode character, `&lt;` or even an imported icon but not a number qualifier